userbot/modules/afk.py

```diff
@@ -54,30 +54,30 @@
     global COUNT_MSG
     if not is_redis_alive():
         return
-    AFK = REDIS.get('isafk')
+    AFK = await is_afk()
     if e.is_private and not (await e.get_sender()).bot:
-        if AFK:
+        if AFK is True:
             if e.sender_id not in USERS:
                 await e.reply(
                     "Sorry! My boss is AFK due to ```"
-                    + AFK
+                    + await afk_reason()
                     + "``` I'll ping him to look into the message soon😉"
                 )
-                USERS.update({sender.sender_id: 1})
+                USERS.update({e.sender_id: 1})
```

Comment:
I would rather replace this with ```sender``` instead of inlining the `sender` to ```e```

```suggestion
                USERS.update({sender.sender_id: 1})
```